spec/statsd_spec.rb

```diff
@@ -1,43 +1,43 @@
-require File.join(File.dirname(__FILE__), "helpers")
-require "sensu/extensions/statsd"
+require File.join(File.dirname(__FILE__), 'helpers')
+require 'sensu/extensions/statsd'
 
-describe "Sensu::Extension::StatsD" do
+describe 'Sensu::Extension::StatsD' do # rubocop:disable Metrics/BlockLength
```

Comment:
I'd be open to disabling this for the whole spec folder as it really would not affect any production code just longer to test code if inefficient.

Reply:
Added a .rubocop.yml to disable the spec folder.